src/content/for_developers/community-membership/_index.en.md

```diff
@@ -195,7 +195,7 @@ The following apply to people who would be an owner:
 [parent process]: https://github.com/kubernetes/community/blob/7d2ebad43cde06607cde3d55e9eed4bb08a286a9/community-membership.md
 [code reviews]: https://github.com/kubernetes/community/blob/7d2ebad43cde06607cde3d55e9eed4bb08a286a9/contributors/guide/collab.md
 [community expectations]: https://github.com/kubernetes/community/blob/7d2ebad43cde06607cde3d55e9eed4bb08a286a9/contributors/guide/expectations.md
-[contributor guide]: https://submariner-io.github.io/contributing/
+[contributor guide]: ../dev_guide
```

Comment:
@dfarrell07 I assume this is correct? Also perhaps we should call it "developer guide"?

Reply:
Yep, I have these in https://github.com/submariner-io/submariner-website/pull/389/commits/42859aa5f03d60b1a6e9df11df7e502924c2cff4 but it's based on https://github.com/submariner-io/submariner-website/pull/388 which needs attetion.

Reply:
ahh - you're already on it - I'll close this.